subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -598,12 +598,44 @@ config BT_CTLR_DYNAMIC_INTERRUPTS
 	  permit use of SoC's peripheral for custom use when Bluetooth is not
 	  enabled.
 
+choice
+	prompt "Optimization options"
+	depends on !LTO
+	default BT_CTLR_OPTIMIZE_FOR_SPEED
```

Comment:
Previously we defaulted to `default y if BT_CTLR_LE_ENC` but now it's always the default. Is that intentional?

Reply:
yes, intentional. This is leftover from legacy LL implementation, this is not a requirement upstream now.